packages/components/src/local/mapping/index.stories.tsx

```diff
@@ -243,3 +244,44 @@ WithPhases.story = {
     }
   }
 }
+
+/**
+ * VIEW WITH ASSET ROUTES
+ */
+const trimmedLabel = 'Trimmed'
+const trimmedDefaultValue = false
+const selectedLabel = 'Selected'
+const selectedDefaultValue = false
+
+
+const greenForce: any = forces[3]
+const platform: any = greenForce.assets[0]
+const planned = platform.plannedTurns
```

Comment:
```suggestion
const  { planned: plannedTurns, history }  = platform
```

Pretty sure that will work, ignore if it I'm wrong though. Is there a reason you had to rename the variable? If not then the following would be preferable. 

```suggestion 
const { plannedTurns, history } = platform
```